TOPSPIN/Phase_Correction/3D.md

```diff
@@ -94,16 +86,22 @@ only `PHC0`, not `PHC1`.
    
 ![after phasing](images/3D_phased_1.FID.png)
 
-7. **Save and Apply PHC0:**  
+4. **Save and Apply PHC0:**  
   - Note down the optimal `PHC0` value (e.g., `151.40` degrees). Click the **Save-disk nD** icon to copy the values to 
 the 3D spectrum, then click the **Return icon**. Double-click the processed 3D spectrum and ensure all `PHC0` and 
 `PHC1` values except for **F3(PHC0)** are still set to `0`.
    
 ![correct phase parameters](images/3D_correct_PROCPARS.png)
 
-8. **Repeat NUS Reconstruction:**  
-  - If NUS was used, follow [this tutorial](../NUS_reconstruction/3D.md) to repeat NUS-reconstruction with the correct 
-phase angles, otherwise just run `ft3d`.
+5. **Initiate NUS-reconstruction:**
+  - Execute `FnTYPE`. If this option is **not** set to `non-uniform sampling` then you can safely use `ftnd 0 nusft` or just `ft3d`
+instead of `ftnd 0` to skip the NUS-reconstruction in all relevant steps from this point on. But even if you run `ftnd 0`, 
+Topspin will recognize it by itself and skip the otherwise tedious NUS-reconstruction. 
+  - Select the `cc` method within the **NUS** section of **PROCPARS** window or simply execute `3 Mdd_mod c`.
```

Comment:
Here are two typos: it is `cs` mode and the command would be `3 Mdd_mod cs` 

It is btw really the best mode for the 3D reconstruction?

Reply:
yes, cs is the best according to Petr Padrta.